geoportailv3.mk

```diff
@@ -17,6 +17,8 @@ DISABLE_BUILD_RULES = test-packages test-packages-ngeo
 
 CONFIG_VARS += ldap
 
+UTILITY_HELP = "- update-translations		Syncronise the localisation with Transifex"
```

Comment:
Syncronise the localisation with Transifex -> Synchronize the translations with Transifex